ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -253,6 +263,14 @@ public PendingTransactions getPendingTransactions() {
       return basicValidationResult;
     }
 
+    if (isLocal
+        && configuration.getStrictTransactionReplayProtectionEnabled()
```

Comment:
Encapsulating a few of these conditions into a method would make this easier to read. Perhaps we can identify the ones that represent the besu client configuration as enabling replay protection, and group them together?

Reply:
sure - sounds like a good idea 👍

Reply:
@jflo - updated and ready for a review if you have time :)